webpack.config.js

```diff
@@ -130,6 +130,40 @@ module.exports = {
         // match the requirements. When no loader matches it will fall
         // back to the "file" loader at the end of the loader list.
         oneOf: [
```

Comment:
Looks like we can achieve multiple sprites through multiple configurations with different `include` paths and [named files extraction](https://github.com/kisenka/svg-sprite-loader#sprite-filename).

But I'm not sure. Anyway, we don't need it right now – it's just one of possible solutions for our future nonexistent problem. 😉